src/ngx_http_lua_worker.c

```diff
@@ -107,6 +107,7 @@ ngx_http_lua_ffi_enable_privileged_agent(char **err)
                                            ngx_core_module);
 
     ccf->privileged_agent = 1;
+    ccf->connection_n = connections;
```

Comment:
we should not overwrite the existing `connection_n`.

Reply:
why this is not privileged_agent_connections